examples/pytorch/llm/llm_infer.py

```diff
@@ -76,18 +76,16 @@ def llm_infer(args: InferArguments) -> None:
             bnb_4bit_quant_type=args.bnb_4bit_quant_type,
             bnb_4bit_use_double_quant=args.bnb_4bit_use_double_quant)
         logger.info(f'quantization_config: {quantization_config.__dict__}')
-        model_kwargs['quantization_config'] = quantization_config
+        kwargs['quantization_config'] = quantization_config
+
+    if args.sft_type == 'full':
+        kwargs['model_dir'] = args.ckpt_dir
     model, tokenizer = get_model_tokenizer(
-        args.model_type, torch_dtype=args.torch_dtype, **model_kwargs)
+        args.model_type, torch_dtype=args.torch_dtype, **kwargs)
 
     # ### Preparing lora
     if args.sft_type == 'lora':
         model = Swift.from_pretrained(model, args.ckpt_dir)
-    elif args.sft_type == 'full':
```

Comment:
How to load checkpoint if this `full` sft_type branch is removed?

Reply:
We pass model_dir=ckpt_dir into get_model_tokenizer